ims/einstein/ceph.py

```diff
@@ -117,6 +129,22 @@ def clone(self, parent_img_name, parent_snap_name, clone_img_name):
             raise file_system_exceptions.ArgumentsOutOfRangeException()
 
     @log
+    def list_children(self, img_id, parent_snap):
+        """
```

Comment:
> all methods talking to ceph or any file system should be grouped together (that is the driver). I have added a docstring that this method is only needed by tests.

No need, sometimes, tests particularly integration/e2e's will have certain auxilary fn's like bringing up a particular component(like ceph) etc. There is no need to have function in main file to be used to in tests in such scenarios. 

> If we support another file system, then this method (an abstract version of course) can be a part of the file system interface because we would require it to be implemented for our tests.

Again this is debatable. Some filesystems may not have a list_snapshot functionality..

That being said, since this is a philosophical question, and I don't want to this to be blocked on small things, I am fine with having a clear comment saying that this only being used it tests.